src/Storages/System/StorageSystemPartsBase.cpp

```diff
@@ -239,7 +243,8 @@ Pipe StorageSystemPartsBase::read(
     const size_t /*max_block_size*/,
     const unsigned /*num_streams*/)
 {
-    bool has_state_column = hasStateColumn(column_names, metadata_snapshot);
+    bool has_state_column = hasVirtualColumn("_state", column_names, metadata_snapshot);
```

Comment:
But why this column is virtual?

Reply:
I guess because by default only parts with committed/outdated are returned - https://github.com/ClickHouse/ClickHouse/blob/master/src/Storages/System/StorageSystemPartsBase.cpp#L55

Reply:
Ok.

But is there a reason to make `use_count` virtual?

Reply:
The only reason for me is that it is pretty internal thing (and some users may be confused with use_count always >= 2), but I'm fine to make non-virtual